templates/cell_neighborhood_diversity.ipynb

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
<p>Should also add a reference to neighborhood mat creation, and probably check within the function that the file exists. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/989/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>